### DISCUSSION TITLE
File chat.conf?

### DISCUSSION THREAD

**Discussion by tuFFo126** [knrd1/chatgpt#39] [Q&A] [answered]
How to set the configuration file for a free account on OpenAI? For model gpt-3.5-turbo 3(RPM) 200(RPD) 40,000(TPM).
It connects, but I get the message when I try to talk to my chatbot: 
"API call failed. You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors."

[chatcompletion]
model = gpt-3.5-turbo
role = user
context =
temperature = 0.8
max_tokens = 200
top_p = 1
frequency_penalty = 3
presence_penalty = 3
request_timeout = 60

What configuration entries should I make?

**Comment by knrd1** [comment, marked as answer]
Hey, the chat.conf file is OK. Also, message seems to be clear, please check your usage by visiting https://platform.openai.com/usage

I can see two options:

1) Free trial credit has expired after three months you first created your OpenAI account.
2) You exhausted your free limit for your trial account within three month period.
